wooting-macro-backend/src/lib.rs

```diff
@@ -263,7 +279,10 @@ async fn execute_macro(macros: Macro, channel: UnboundedSender<rdev::EventType>)
             let cloned_channel = channel;
 
             task::spawn(async move {
-                macros.execute(cloned_channel).await;
+                macros
+                    .execute(cloned_channel)
+                    .await
+                    .unwrap_or_else(|err| error!("Error executing macro: {}", err));
```

Comment:
`if let Err...` would be a bit cleaner here

Reply:
Fixed: [d016293](https://github.com/WootingKb/wooting-macros/pull/194/commits/d0162937975c97aaf54fa662b69dc541e1645b6e)